src/meta/meta_bulk_load_service.cpp

```diff
@@ -1408,10 +1466,11 @@ void bulk_load_service::on_query_bulk_load_status(query_bulk_load_rpc rpc)
     }
 
     if (!app->is_bulk_loading) {
-        auto hint_msg = fmt::format("app({}) is not during bulk load", app_name);
+        auto hint_msg = fmt::format("app({}) is not during bulk load, return last time result", app_name);
         derror_f("{}", hint_msg);
-        response.err = ERR_INVALID_STATE;
+        response.err = ERR_OK;
         response.__set_hint_msg(hint_msg);
+        response.__set_last_time_result(app->last_time_result);
```

Comment:
query_bulk_load_response doesn't have a filed called `last_time_result`, and this pull request doesn't adds the filed for this struct.

Reply:
Added in  src/common/bulk_load.thrift